src/frontend/data/sideEffects/updateLiveParticipants/index.tsx

```diff
@@ -0,0 +1,82 @@
+import { appData } from 'data/appData';
+import { API_ENDPOINT } from 'settings';
+import { Participant } from 'types/Participant';
+import { Video } from 'types/tracks';
+
+const updateLiveParticipants = async (
+  video: Video,
+  participant: Participant,
+  participantList: 'participants-asking-to-join' | 'participants-in-discussion',
+  method: 'POST' | 'DELETE',
+  errorMessage: string,
+): Promise<Video> => {
+  const response = await fetch(
+    `${API_ENDPOINT}/videos/${video.id}/${participantList}/`,
+    {
+      body: JSON.stringify(participant),
+      headers: {
+        Authorization: `Bearer ${appData.jwt}`,
+        'Content-Type': 'application/json',
+      },
+      method,
+    },
+  );
+
+  if (!response.ok) {
+    throw new Error(`${errorMessage} ${video.id}.`);
+  }
+
+  return response.json();
+};
+
+export const addParticipantAskingToJoin = async (
+  video: Video,
+  participant: Participant,
+): Promise<Video> => {
+  return updateLiveParticipants(
+    video,
+    participant,
+    'participants-asking-to-join',
+    'POST',
+    'Failed to add a participant asking to join the live video',
```

Comment:
Shouldn't we handle internationalization for that ?

Reply:
It looks like we don't translate messages in data/sideEffects

Reply:
According to the discussion we had earlier today with @lunika this shouldn't be about anonymous-id but displayed name since we should prompt displayed name form for anonymous user before they can use the 'on stage' feature, isn't it ?